src/resources/elements/lazylinkfield.js

```diff
@@ -119,4 +119,16 @@ export class LazyLinkfield extends Field {
   getValue() {
     return this.child ? this.child.getValue() : undefined;
   }
+
+  resolvePath(path) {
+    const superResolv = super.resolvePath(path);
```

Comment:
Try not to use abbreviated words. Instead of `resolv`, spell `resolve` fully. 

See:

- Clean Code JavaScript: [Avoid Mental Mapping](https://github.com/ryanmcdermott/clean-code-javascript#avoid-mental-mapping)